removal_from_ansible.rst

```diff
@@ -0,0 +1,160 @@
+*****************************************************
+Ansible Community Package Collections Removal Process
+*****************************************************
+
+.. contents:: Topics
+
+Overview
+========
+
+Sometimes the Ansible community removes a collection from the Ansible package. We try to do this only rarely. This document describes why we might remove a collection from the `Ansible community package <https://pypi.org/project/ansible/>`_ (`build data <https://github.com/ansible-community/ansible-build-data/>`_).
```

Comment:
I think it's a good idea to keep this, at least for now, so that nobody gets the impression that we're just waiting to kick out tons of stuff :) Removing right now is a last resort if nothing else worked, and I think it should stay that way. But this is definitely something to discuss, maybe let's keep this for now (unresolved) and see if others chime in as well.

Reply:
Though maybe something like "We would like to avoid this whenever possible" or something like that?

Reply:
> Though maybe something like "We would like to avoid this whenever possible" or something like that?

I'm not sure, sounds a bit apologetic to me. Like "Sorry for you inconvenience". But maybe "removes a collection from the Ansible package for stability or security reasons" would be a good wording. I think this would (implicitly) make clear that it's not done lightly, but would also make clear _why_ it's sometimes done in the very first sentence of the document.

Reply:
I would also add 'legal' here, since that would also be a valid reason to remove something. How about the following?
```suggestion
Sometimes the Ansible community removes a collection from the Ansible package for stability, legal, or security reasons. This document describes why we might remove a collection from the `Ansible community package <https://pypi.org/project/ansible/>`_ (`build data <https://github.com/ansible-community/ansible-build-data/>`_).
```

Reply:
+1 for this way of expressing it.